### PR TITLE
Support a binder badge

### PR DESCRIPTION
**Pull request by uribo**
My Proposal is here #239 

Added new function `use_binder_badge()` that prints out [Binder](https://mybinder.org/) badge.

### CODE REVIEW

File: R/badge.R [hunk 1]
```diff
@@ -78,6 +78,32 @@ stages <- c(
   questioning = "blue"
 )
 
+#' Create a Binder badge
+#'
+#' This prints out the markdown which will display a Binder "badge", indicating
+#' launch build your repository source codes on Binder <https://mybinder.org/>.
```
Comment (jennybc) [inline R/badge.R]: This part of the sentence still doesn't make sense to me:

> indicating launch build your repository source codes on Binder

I visited <https://mybinder.org> and tried to understand what's going on. Does this sound correct?

> indicating that your repository can be launched in an executable environment on <https://mybinder.org/>.

If that's OK, let's use that wording (or something similar).

File: R/badge.R [hunk 2]
```diff
@@ -78,6 +78,32 @@ stages <- c(
...
+#'
+#' @export
+use_binder_badge <- function() {
+
+  if (uses_git(path = proj_get()) == TRUE) {
```
Comment (jennybc) [inline R/badge.R]: You don't need the `== TRUE`. `uses_git()` already returns either `TRUE` or `FALSE`.

Also, I wonder if this should be `uses_github()` in this case?

**Comment by jennybc** [comment]
Will you also add a bullet to NEWS.md?

File: R/badge.R (hunk 1 above)
Comment (uribo) [reply R/badge.R]: Thank you for your suggestion. That is the tidy phrase :)

File: R/badge.R (hunk 2 above)
Comment (uribo) [reply R/badge.R]: Hmm. Indeed, I might as well change to `uses_github()` as you say.

[Here](https://github.com/r-lib/usethis/blob/d12b3f14c24768c65aa2abf147f1a2951c500a14/R/github.R#L72), `uses_github()` is used the same way.

**Comment by uribo** [comment]
Done. But I'm sorry, lowered coverage.

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**